### DISCUSSION TITLE
how to set up with babel-plugin-preval?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11628] [Help]

**Comment by wongjn** [comment]
Consider reviewing [the README for `babel-plugin-preval`](https://github.com/kentcdodds/babel-plugin-preval#installation) and also [configuring Babel in Next.js](https://nextjs.org/docs/pages/building-your-application/configuring/babel).

**Reply by anessheremeti** [reply]
Ask chat gpt? Send him your code and tell the error that is showing on console or what should be done instead !

**Comment by anessheremeti** [comment]
give me your telegram or something
